sdcm/nemesis.py

```diff
@@ -600,7 +600,7 @@ def disrupt_nodetool_drain(self):
             self.target_node.start_scylla_server(verify_up=True, verify_down=False)
 
     def disrupt_ldap_connection_toggle(self):
-        self._set_current_disruption('Disconnect and Reconnect LDAP connection')
+        self._set_current_disruption('Disconnect_and_Reconnect_LDAP_connection')
```

Comment:
a nitpick, it might be better that LDAP is at the beginning of name of the nemesis (since it's a long name, and on the dashboards you won't see it all)

Reply:
updated them

Reply:
@fruch ^^